<?xml version="1.0"?>
<launch>
	<!-- acquisition arguments -->
	<arg name="thigh_length" default='0.47'/>
	<arg name="shin_length" default='0.5'/>
	<arg name="front_foot_length" default='0.14'/>
	<arg name="rear_foot_length" default='0.11'/>
	<arg name="foot_width" default='0.1'/>
	<arg name="dist_origin_foot" default='0.05'/>
	<arg name="max_step_length" default='0.7'/>
	<arg name="max_step_height" default='0.4'/>
	<arg name="ema_coeff" default='0.9'/>
	<arg name="step_time" default='2.0'/>
	<arg name="knee_ang" default='15'/>
	
	<node name="cfftg_v2" pkg="cfftg" type="cfftg_v2" output="screen" required="true">
		<param name="~thigh_length" type="double" value="$(arg thigh_length)"/>
		<param name="~shin_length" type="double"  value="$(arg shin_length)"/>
		<param name="~front_foot_length" type="double" value="$(arg front_foot_length)"/>
		<param name="~rear_foot_length" type="double" value="$(arg rear_foot_length)"/>
		<param name="~step_time" type="double"  value="$(arg step_time)"/>
		<param name="~knee_ang" type="double"  value="$(arg knee_ang)"/>
	</node>
	
</launch>
